eng/tools/generator/template/rpName/packageName/autorest.md.tpl

```diff
@@ -9,4 +9,5 @@ require:
 - https://github.com/Azure/azure-rest-api-specs/blob/{{commitID}}/specification/{{rpName}}/resource-manager/readme.go.md
 license-header: MICROSOFT_MIT_NO_VERSION
 module-version: 0.1.0
+{{packageConfig}}
```

Comment:
What does this mean?

Reply:
I have added a config param to release-v2 for multi-package RP's onboard which can be used to add config like `package-singleservers: true`. Also, package config will be extracted from swagger config yaml annotation for SDK automation onboard.